src/iotjs_debuglog.h

```diff
@@ -36,6 +37,8 @@ extern const char* iotjs_debug_prefix[4];
       fprintf(iotjs_log_stream, __VA_ARGS__);                       \
       fprintf(iotjs_log_stream, "\n");                              \
       fflush(iotjs_log_stream);                                     \
+      if(iotjs_debug_extern_out)                                    \
```

Comment:
@daeyeon Please add a space between if and opening parenthesis.

Reply:
fixed.